### DISCUSSION TITLE
How to configure background?

### DISCUSSION THREAD

**Discussion by levino** [storybookjs/react-native#301] [Q&A] [answered]
I do not understand how to load a `preview.js` in version 5.x. Can anyone help? Thanks!

**Comment by dannyhw** [comment, marked as answer]
There is no preview js in the 5.x of storybook. Instead you use the functions like addDecorator and addParameters.

For example with the backgrounds addon you can do:

```
import { getStorybookUI, configure, addParameters, addDecorator } from '@storybook/react-native';
import './rn-addons';
import { withBackgrounds } from '@storybook/addon-ondevice-backgrounds';

addDecorator(withBackgrounds);

addParameters({
  backgrounds: [
    { name: 'dark', value: '#222222' },
    { name: 'white', value: '#ffffff', default: true },
  ],
});
```

you can find this example in the 5.3 example project here
https://github.com/storybookjs/react-native/blob/v5.3.25/examples/native/storybook/index.js

**Reply by levino** [reply]
Thanks!
